style.css

```diff
@@ -63,16 +68,166 @@
   font-size: 16px;
   line-height: 24px;
   text-align: left;
-  padding-left: 5%;
   padding-top: 5%;
+  background: transparent;
 }
 
 .intro-section__social {
   display: flex;
   flex-direction: row;
-  margin: 4%;
+  padding: 4% 4% 4% 0;
+  background: transparent;
 }
 
-.intro-section__social > * {
+.intro-section__social > li {
   padding: 2%;
+  background: transparent;
+}
+
+.project__section {
+  display: flex;
+  flex-direction: column;
+  padding: 5rem 0 5rem 0;
+}
+
+.project__card {
+  margin: 5%;
+  padding: 5%;
+  border: solid 1px #dfe1e6;
+  border-radius: 1rem;
+  background: #fff;
+}
+
+.card__img {
+  width: 100%;
+  height: auto;
+  background: #fff;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: left;
+  background: #fff;
+}
+
+.card__details {
+  display: flex;
+  flex-direction: row;
+  margin: 0 5% 2% 0;
+  background: #fff;
+}
+
+.card__details > * {
+  padding-right: 3.5%;
+  background: #fff;
+}
+
+.details__title {
+  color: #344563;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  background: #fff;
+}
+
+.details__text {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  background: #fff;
+}
+
+.card__description {
+  color: #344563;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  text-align: left;
+  background: #fff;
+}
+
+.tag__parent {
+  display: flex;
+  flex-direction: row;
+  margin: 5% 0 5% 0;
+  background: #fff;
+}
+
+.social {
+  background: transparent;
+}
+
+.tag__parent > * {
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background: #ebedff;
+  border-radius: 0.5rem;
+  margin: 0 3% 0 0;
+  padding: 1.5% 4% 1.5% 4%;
+}
+
+.button__std {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border: 1px solid #6070ff;
+  border-radius: 0.5rem;
+  background: #fff;
+  padding: 0.5rem;
+}
```

Comment:
Great job applying styles to your project buttons ✔️. However, I noticed that it is centered but in the Figma design, they are positioned on the left side. 🤓 Please kindly position it as similar as possible to the Figma design. 🔧

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/235281847-c36355e1-847b-4139-8e36-3fa734d79453.png)


This is your design 👇🏽(almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/235281840-a82ce976-efc3-4deb-a77b-359c57ca583a.png)